index.html

```diff
@@ -184,6 +184,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="?url=https://Robotec-CoderZ.github.io/he.js" data-action="load-url">
+                            <header>
+                                <h2>EV3 LEGO Education</h2>
```

Comment:
I find it confusing that we now have two extensions that at a surface level appear to do the same thing. I recommend that we do the following:

- Rename the existing "EV3+Scratch" extension to "LEGO Mindstorms EV3 (Bluetooth)"
- Rename this extension to "LEGO Mindstorms EV3 (USB)"